supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,30 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[command-line-commands-that-require-elevated-privileges]]
+== Command-line commands that require elevated privileges
+
+Some command-line interface (CLI) commands require that a user run them with "elevated privileges".  
+That is, some commands require that a normal user switch to another user account whose privileges permit certain operations or require that a normal user temporarily change their current privileges without changing user accounts.
+
+=== Guidelines for documenting CLI commands with elevated privileges
+
+When documenting CLI commands that require elevated privileges, consider the following guidelines and formulate a consistent approach to use throughout your content:
+
+* If you include a shell prompt in a sample command, always show the correct prompt for a normal user (`$`) or a user with elevated privileges (`#`).
+* Avoid relying solely on a shell prompt in a sample command to indicate the required privilege level to run a command. 
+If a command or series of commands requires elevated privileges, add phrasing to the introductory sentences to indicate that this is the case. For example, "Connect to the cluster node and become the root user".
++
+[NOTE]
+====
+Avoid using variations such as "become root" or similar phrasing.
+====
++
+* If a single command requires temporarily elevated privileges, use `sudo` in the sample command. Do not use `su` without good cause.
+* If only a few privileged commands occur in a mostly unprivileged series of commands, use `sudo` on a per-command basis. 
+Do not rely solely on changes of the shell prompt in the sample syntax to indicate the required permission level.
```

Comment:
Add the following sentence as an admonition? I think it reads like one, because it does not directly align with the two previous "if" statements. 

> Do not rely solely on changes of the shell prompt in the sample syntax to indicate the required permission level.
-------------------------------------------------------
I've had to read the following sentence a few times, so that I could understand it:

> If only a few privileged commands occur in a mostly unprivileged series of commands, use `sudo` on a per-command basis. 

Privileged commands = sudo-level commands?
Unprivileged commands = regular-user commands?

Reply:
You're right that's confusing. I've reorganized and rewritten the bulleted guidelines.